openslides_backend/action/actions/meeting_user/create.py

```diff
@@ -38,3 +40,33 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 f"MeetingUser instance with user {instance['user_id']} and meeting {instance['meeting_id']} already exists"
             )
         return super().update_instance(instance)
+
+    def get_history_information(self) -> Optional[HistoryInformation]:
+        information = {}
+        for instance in self.instances:
+            instance_information = []
+            if "group_ids" in instance:
+                if len(instance["group_ids"]) == 1:
+                    instance_information.extend(
+                        [
+                            "Participant added to group {} in meeting {}",
+                            fqid_from_collection_and_id(
+                                "group", instance["group_ids"][0]
+                            ),
+                        ]
+                    )
+                else:
+                    instance_information.append(
+                        "Participant added to multiple groups in meeting {}",
+                    )
+            else:
+                instance_information.append(
+                    "Participant added to meeting {}",
```

Comment:
Our definition of user in meeting is the membership in a group of the meeting. 
This case here has no description as far as I know

Reply:
Yes, that stems form the fact that you can create a meeting user without passing any groups. This means he was (somewhat) added to the meeting, but without groups. In practice, this does not make sense - maybe `group_ids` should be required? This would ofc require new changes...